junebug/workers.py

```diff
@@ -201,14 +201,47 @@ def _forward_event_http(self, event):
         if url is None:
             return
 
+        # Parse the basic auth username & password if available
+        url_components = urlparse(url)
+        username = url_components.username
+        password = url_components.password
+        if any([username, password]):
+            url = urlunparse((
+                url_components.scheme,
+                '%s%s' % (
+                    url_components.hostname,
+                    (':%s' % (url_components.port,)
+                     if url_components.port is not None
+                     else '')),
+                url_components.path,
+                url_components.params,
+                url_components.query,
+                url_components.fragment,
+            ))
+            auth = (username, password)
+        else:
+            auth = None
+
+        # Construct token auth headers if configured
+        auth_token = yield self._get_event_auth_token(event)
+
+        if auth_token:
+            headers = {
+                'Authorization': [
+                    'Token %s' % (auth_token,)]
+            }
+        else:
+            headers = {}
+
```

Comment:
Do we perhaps want to remove the duplication of parsing auth creds from the url here? Perhaps a shared function for both mo & events since they're largely doing the same thing here.